Modules/Loadable/Segmentations/Widgets/qMRMLSegmentEditorWidget.cxx

```diff
@@ -1025,15 +1025,15 @@ void qMRMLSegmentEditorWidget::updateEffectList()
 
   if (d->toolButton(nullptr) == nullptr)
     {
-    // Add NULL effect (arrow button to deactivate all effects)
+    // Add nullptr effect (arrow button to deactivate all effects)
     QToolButton* effectButton = new QToolButton(d->EffectsGroupBox);
-    effectButton->setObjectName(NULL_EFFECT_NAME);
+    effectButton->setObjectName(nullptr_EFFECT_NAME);
```

Comment:
Seems incorrect

Reply:
Thanks, fixed the regex (updated the wiki)